charts/azure/azure-disk-loss/rbac.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-disk-loss-sa
+  namespace: default
+  labels:
+    name: azure-disk-loss-sa
+    app.kubernetes.io/part-of: litmus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: azure-disk-loss-sa
+  namespace: default
+  labels:
+    name: azure-disk-loss-sa
+    app.kubernetes.io/part-of: litmus
+rules:
+  - apiGroups: ["", "batch", "litmuschaos.io"]
+    resources: ["jobs", "pods", "pods/log", "events", "pods/exec", "chaosengines", "chaosexperiments", "chaosresults", "secrets"]
+    verbs: ["create", "list", "get", "patch", "update", "delete"] 
```

Comment:
can we break this based on apiGroups like https://github.com/litmuschaos/chaos-charts/blob/d41eecd02fabac9f56e9c421cc3e7e8b2fc0c15e/charts/aws-ssm/aws-ssm-chaos-by-tag/rbac.yaml#L18